CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ BUG FIXES:
 
 * `resource/kubernetes_stateful_set`: Fix updates of stateful set images ([#252](https://github.com/terraform-providers/terraform-provider-kubernetes/issues/252))
 
+FEATURES:
```

Comment:
@alexsomesan I should probably have put `FEATURES` before `IMPROVEMENTS` and `BUG FIXES`